packages/main/src/Popover.hbs

```diff
@@ -1,17 +1,27 @@
-{{>include "./Popup.hbs"}}
+<div class="ui5-popover-root">
```

Comment:
Don`t forget to add role, aria-modal and aria-labelledby

Reply:
I totally forgot about the acc, I will add the required labels and stuff in the next patch